test/blackbox-tests/test-cases/interfere-sandbox-deletion.t

```diff
@@ -22,10 +22,17 @@ the rule and the sandbox cleanup.
   3 |  (action (system "\| touch foo && mkdir bar && touch bar/x && chmod -w bar &&
   4 |                  "\| echo failed action && exit 1
   5 |           )))
-  failed action
-  Error:
+  Error: failed to delete sandbox in
+  .sandbox/$SANDBOX
+  Reason:
   .sandbox/$SANDBOX/default/bar): Directory not empty
```

Comment:
Where is this unmatched closing parenthesis coming from? Can we drop it?

Reply:
It's just the `sed` that removes the sandbox id misbehaving. Fixed.